spec/features/publish/cookie_preferences_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+feature "Updating cookie preferences" do
+  scenario "i can update my cookie preferences" do
+    when_i_visit_the_cookie_preferences_page
+    and_i_give_consent_and_submit
+    then_i_should_see_a_confirmation_message
```

Comment:
Do you reckon its worth adding in another method to check that you can successfully opt out of cookies, i.e select "No"?

Reply:
Probably not as it's still the same updating logic (just inverse of values being set)